packages/airnode-deployer/src/cli/commands.test.ts

```diff
@@ -0,0 +1,124 @@
+import { join } from 'path';
+import { mockReadFileSync } from '../../test/mock-utils';
+import { readFileSync } from 'fs';
+import { deploy, remove, removeWithReceipt } from './commands';
+import { version as packageVersion } from '../../package.json';
+import { Receipt } from '../types';
+import * as logger from '../utils/logger';
+
+const readExampleConfig = () => JSON.parse(readFileSync(join(__dirname, '../../config/config.example.json'), 'utf-8'));
+
+jest.mock('../infrastructure', () => ({
```

Comment:
FYI: There are some non trivial things happening during mocking. 

These mocks are hoisted before the imports when the tests are run. However, if you use:
```js
const mockVariable = jest.fn()
jest.mock('../module', () => ({
    val: mockVariable
}))
```

this won't work unless you explicitely move the imports below. The variables are hoisted as well (the `const mockVariable` part), but the initialization seems not. 

This way it works a bit better. All imports are written first, which feels natural and inside code you get the mocked functions to make assertions.